src/rdkafka.h

```diff
@@ -3920,7 +3920,12 @@ rd_kafka_consumer_group_metadata_read (
  *                               partition, either set manually or by the
  *                               configured partitioner.
  *
- *    .._F_FREE and .._F_COPY are mutually exclusive.
+ *    .._F_FREE and .._F_COPY are mutually exclusive. If neither of these are
+ *    set, the caller must ensure that the memory backing \p payload is not
+ *    reused until the delivery callback is called. Other buffers passed to
```

Comment:
Perhaps "delivery callback is invoked" instead of "delivery callback is called"?

Reply:
Yeah, can we also make it even more clear with something like:
"..backing \p payload remains valid and is not modified or reused until .."